CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # Changelog
 
+### 1.0.11 - Wednesday 8 April 2020
+
+- Users can now change password in /setup
```

Comment:
Missed one of the coolest features which is being able to filter the Trends view by any Event (rather than just actions). #442

Reply:
Will update this and array